AsyncDisplayKit/ASCellNode.h

```diff
@@ -16,6 +16,7 @@ typedef NSUInteger ASCellNodeAnimation;
 
 @protocol ASCellNodeLayoutDelegate <NSObject>
 
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView cellFrameInScrollView:(CGRect)cellFrame;
```

Comment:
Because this is a delegate, we need to send the cell too.

Let's do visibleNode:(ASCellNode *)node scrollViewDidScroll:(UIScrollView *)scrollView withCellFrameInScrollView:(CGRect)cellFrame